frontend/src/views/ProprietairePage.vue

```diff
@@ -1517,7 +1517,7 @@ export default defineComponent({
 }
 
 .required-label {
-  color: red;
+  color: #e1000f;
```

Comment:
Au mieux il faut utiliser les varibles si possible, tu peux essayer avec ca

```suggestion
  color: var(--red-marianne-main-472);
```